__init__.py

```diff
@@ -426,6 +458,8 @@ def draw_push_to_talk_button(self, context):
     layout = self.layout
     layout.enabled = os_platform in supported_platforms
 
+    layout.separator_spacer()
```

Comment:
For now please do not add this. Moving the button to the side makes is difficult to interact with.

Reply:
Removed in the updated patch.